### PR TITLE
clean up tips and tricks page

### PR DESCRIPTION
**Pull request by prashn64**

### CODE REVIEW

File: guides/tips-and-tricks.md [hunk 1]
```diff
@@ -6,7 +6,13 @@ permalink: /guides/tips-and-tricks/
...
-* You can run this <a href="https://github.com/smfoote/Swiffer.js" target="_blank">Dust linter</a> to get warned while you are writing templates about possible security holes, parse errors, coding style guidelines, etc.
\ No newline at end of file
+* You can run this <a href="https://github.com/smfoote/Swiffer.js" target="_blank">Dust linter</a> to get warned while you are writing templates about possible security holes, parse errors, coding style guidelines, etc.
+
+### Debugging
+Debugging can be turned on by setting dust.debugLevel to one of 4 values: "DEBUG", "INFO", "WARN", "ERROR" before your render/renderSource/stream calls.  "DEBUG" will not filter out any log messages, while every level above will filter out logs below them in severity.  It is useful to set dust.debugLevel to "DEBUG" in your testing environment, and either to "ERROR" or unset in your production environment.
```
Comment (sethkinast) [inline guides/tips-and-tricks.md]: Can you put stuff in `backticks` like `dust.debugLevel`

File: guides/tips-and-tricks.md [hunk 2]
```diff
@@ -6,7 +6,13 @@ permalink: /guides/tips-and-tricks/
 
 # Tips and Tricks
 
+
+### Design Philosophy
+Although Dust is not completely logicless, so that you can get around some basic hurdles, it is still wise not to bloat templates with too much "business logic."  Business logic is essentially the way the data is stored.  If there's too much logic in a Dust template, it will be hard to read and hard to maintain.  Additionaly, you will eventually hit up against a wall with the amount of logic that Dust can handle, and will end up writing an endless number of helper functions to shoehorn your use case in.  This will just add to the difficulties in maintaining the template.  Let the server handle the business logic.
```
Comment (smfoote) [inline guides/tips-and-tricks.md]: Is this the right place for this paragraph? Also, it seems like we could shorten it to say "Don't put business logic in your template, even if it's possible."

Comment (prashn64) [inline guides/tips-and-tricks.md]: Closest thing I could think of for a tip.  Do you mean shorten the entire paragraph?

Comment (smfoote) [inline guides/tips-and-tricks.md]: Yeah, I think concise is better, especially in documentation.

**Comment by smfoote** [comment]
Do a `pull` and I'll merge this.

**Pull request closed without merging**